### ISSUE TITLE
Countdown syntax (?) issues and use of YAML frontmatter dates

### ISSUE THREAD

**Issue by merlinuwe** [rafaelveiga/obsidian-widgets#6]
What am I doing wrong?

The first example calculates reverse,

the second example (2025), too.

The last calculation is correct.

![image](https://github.com/rafaelveiga/obsidian-widgets/assets/4138869/fb9a1ebc-7877-4d43-98f7-23faa9999b00)

Am I missing something?

---

In YAML frontmatter I use some dates, for example:

```text
erstellt: Sonntag, 28. Januar 2024, 18:43:32 Uhr
geändert: Sonntag, 28. Januar 2024, 18:45:58 Uhr
Termin: 2023-09-19T17:00:00
Termin2: 2024-01-30 00:00:00
```

Is it possible, to use these dates in your smart widget?

**Comment by rafaelveiga** [comment]
Hey @merlinuwe, thanks for the feedback and for using the plugin! 

In the first example, the count is correct because the date is in the past. I plan to fix this issue by not rendering the countdown if the date is in the past. 

In the second example, you switched the property `to` with `date`, thus, rendering the date in example 1 in the countdown. If you do the following, it should work:

````
```widgets
type: countdown
date: 2025-01-01 00:00:00
to: 2025-01-01 00:00:00
```
````

---
About using dates in the YAML frontmatter as source for the countdown, that is a great idea! I'm going to add it to our roadmap

**Comment by rafaelveiga** [comment]
I've also noticed a bug in your screenshot. The `to` property is `2025-01-01 00:00:00` but the widget only renders `2025-01-01 00`. Added this to our bug tracker

**Comment by rafaelveiga** [comment]
Most of this was fixed on release [0.0.6](https://github.com/rafaelveiga/obsidian-widgets/releases/tag/0.0.6).
